Astro 8/31 release notes (#1115)
astronomer/docs: astro/deployment-metrics.md

![DAGs page](/img/docs/dags-page.png)

## Export Airflow metrics to Datadog

> You can export metrics about a Deployment's Airflow environment to Datadog. The Datadog Agent collects metrics that are available in the Astro UI as well as more specific metrics about your environment's performance. See the [Datadog documentation](https://docs.datadoghq.com/integrations/airflow/?tab=host#data-collected) for a complete list of exported metrics.

Reviewer: ```suggestion
You can now export Airflow Deployment metrics to Datadog. The Datadog Agent collects metrics that are available in the Astro UI and environment performance metrics. See the [Datadog documentation](https://docs.datadoghq.com/integrations/airflow/?tab=host#data-collected) for a complete list of exported metrics.
```

Author: This edit makes the feature sound automatic, when the exporting must be manually configured

1. Create a new Datadog API key or copy an existing API key. See [Datadog documentation](https://docs.datadoghq.com/account_management/api-app-keys/).
2. [Create a new environment variable](environment-variable.md) in your Deployment. Specify the variable key as `DATADOG_API_KEY` and its value as your Datadog API key. Select the **Secret** checkbox.
3. Save the environment variable.

> After you configure the environment variable, Astro launches a sidecar container in your Deployment that runs [DogStatsD](https://docs.datadoghq.com/developers/dogstatsd/?tab=hostagent). This container works with your Deployment's existing StatsD infrastructure to export Airflow metrics to the Datadog instance associated with your API key.

Reviewer 2: Please add a note at the end of this section, specifying that this integration will NOT be collecting `service checks` on the Airflow instance (https://docs.datadoghq.com/integrations/airflow/?tab=host#service-checks), so users won't be able to see them.
